### PR TITLE
changed crate from `vst2` to `vst`

### PR DESCRIPTION
**Pull request by piedoom**
In preparation to solve issue #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ crate-type = ["cdylib"]
 [trav-url]: https://travis-ci.org/rust-dsp/rust-vst
 [appv-img]: https://ci.appveyor.com/api/projects/status/npiyjfithlx50hfs?svg=true
 [appv-url]: https://ci.appveyor.com/project/rustdsp/rust-vst
-[crates-img]: https://img.shields.io/crates/v/vst2.svg
-[crates-url]: https://crates.io/crates/vst2
+[crates-img]: https://img.shields.io/crates/v/vst.svg
```
Comment (piedoom) [inline README.md]: I don't believe we're currently using a few images but names were changed just in case

**Comment by piedoom** [comment]
Please note that the authors section in `Cargo.toml` has been updated.  I have filled in each of the [contributors](https://github.com/rust-dsp/rust-vst/graphs/contributors).  

Please look this over and make sure you are addressed by the correct name.  Out of the sake of privacy, I have not included email addresses.  Please respond to have any changes, to add your mailing address, or to request that your name be removed.

 I apologize for the tagging spam, but I'd rather be safe than sorry!  @overdrivenpotato @Boscop @zyvitski @Ms2ger @robsaunders @conundrumer @askeksa @kykc @Earlz @feyl @XMPPwocky @Wallacoloo and @henrikno, please make sure the names listed are correct.

**Comment by Ms2ger** [comment]
I don't recall touching this crate, but sure :)

**Comment by piedoom** [comment]
🎊

**Pull request merged**